perma_web/perma/templates/user_management/includes/created-link-items.html

```diff
@@ -1,8 +1,4 @@
-{% load local_datetime pipeline %}
-
-{% if in_iframe %}
-    {% stylesheet 'base' %}
-{% endif %}
```

Comment:
The whole `folder_contents` view seems to be an unmaintained fallback for non-JS browsers that click the folder tree: https://perma-stage.org/manage/links/folder/24?iframe=1

Possibly that whole thing should just be ripped out?